android-components/samples/browser/src/main/java/org/mozilla/samples/browser/BaseBrowserFragment.kt

```diff
@@ -74,6 +74,7 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Activit
         _binding = FragmentBrowserBinding.inflate(inflater, container, false)
 
         binding.toolbar.display.menuBuilder = components.menuBuilder
+        binding.engineView.setActivityContext(requireActivity() as BrowserActivity)
```

Comment:
Why do we need to cast it to a `BrowserActivity`?

Reply:
Oops, that is not needed - updated.